lib/Cake/View/Helper/PaginatorHelper.php

```diff
@@ -988,17 +988,18 @@ public function last($last = 'last >>', $options = array()) {
 	public function meta($options = array()) {
 		$model = isset($options['model']) ? $options['model'] : null;
 		$params = $this->params($model);
+		$urlOptions = isset($this->options['url']) ? $this->options['url'] : array();
```

Comment:
I would have expected these parameters to be `$params['options']`. Perhaps instead of this change it is $this->url() that needs to be modified?